.murdock

```diff
@@ -5,7 +5,19 @@
 # and / or
 #export APPS="examples/hello-world tests/unittests"
 
-QUICKBUILD_BOARDS="nrf52840dk samr21-xpro esp32-wroom-32 native"
+QUICKBUILD_BOARDS="
+atmega256rfr2-xpro
+esp32-wroom-32
+esp32s3-devkit
+frdm-k64f
+hifive1b
+msb-430
+native
+nrf52840dk
+qn9080dk
+samr21-xpro
+stk3200
+stm32f429i-disc1"
```

Comment:
```suggestion
stm32f429i-disco"
```

This would include a board with CDC ACM stdio.
We could also add `msba2` for an ARM7 board.

Reply:
Agreed, I was also thinking at the adafruit edgebadge which is samd51 with CDC ACM stdio.